testsuite/shared/src/main/java/org/jboss/as/test/shared/util/DisableInvocationTestUtil.java

```diff
@@ -1,10 +0,0 @@
-package org.jboss.as.test.shared.util;
```

Comment:
We still need to keep this for now (once this PR and https://github.com/wildfly/wildfly/pull/10138 are merged, there will still be 2 disabled tests that use this utility).